src/main/java/com/minecolonies/coremod/util/constants/ToolType.java

```diff
@@ -0,0 +1,56 @@
+package com.minecolonies.coremod.util.constants;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public enum ToolType implements IToolType
+{
+    NONE       ("",        false),
+    PICKAXE    ("pickaxe", true),
+    SHOVEL     ("shovel",  true),
+    AXE        ("axe",     true),
+    HOE        ("hoe",     true),
+    SWORD      ("weapon",  true),
+    BOW        ("bow",     false),
+    FISHINGROD ("rod",     false),
+    SHEARS     ("shears",  false),
+    SHIELD     ("shield",  false);
+
+    static final private Map<String,IToolType> tools = new HashMap<>();
+    private final String name;
+    private final boolean variableMaterials;
+
+    static
+    {
+        for(ToolType type : values())
+        {
+            tools.put(type.getName(), type);
+        }
+    }
+
+    private ToolType(final String name, final boolean variableMaterials)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') "private" is redundant in this context. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2333)